spark-etl/src/main/scala/geotrellis/spark/etl/EtlConf.scala

```diff
@@ -40,4 +45,25 @@ class EtlConf(args: Seq[String]) extends ScallopConf(args){
 
   implicit def crsConverter: ValueConverter[CRS] = singleArgConverter[CRS](CRS.fromName)
   implicit def storageLevelConvert: ValueConverter[StorageLevel] = singleArgConverter[StorageLevel](StorageLevel.fromString)
-}
\ No newline at end of file
+}
+object EtlConf {
+ val layoutSchemeConverter = new ValueConverter[LayoutSchemeProvider] {
+   val validNames = List ("tms", "floating")
+
+   def parse(s : List[(String, List[String])]) = s match {
+     case (_, schemeName :: Nil) :: Nil  if validNames contains schemeName =>
+        Right(Some(schemeName match {
+          case "floating" =>
+            (crs: CRS, tileSize: Int) => FloatingLayoutScheme(tileSize)
+          case "tms" =>
+            (crs: CRS, tileSize: Int) => ZoomedLayoutScheme(crs, tileSize)
+        }))
+     case _ =>
+       Left("wrong arguments format")
```

Comment:
I get this error when I don't provide a layerFormat, even though it seems like it should default.